MapboxNavigation/InstructionPresenter.swift

```diff
@@ -0,0 +1,87 @@
+import UIKit
+import MapboxDirections
+
+class InstructionPresenter {
```

Comment:
👍  Cool. Perhaps an Instructable protocol could be the next step.

Reply:
I'm into the idea -- let's look out for a unified interface when we write/refactor the next view which might otherwise contain logic for setting up & displaying a complex domain object.